megamek/unittests/megamek/common/PlanetaryConditionsTest.java

```diff
@@ -0,0 +1,176 @@
+package megamek.common;
+
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import static org.junit.Assert.*;
+
+public class PlanetaryConditionsTest {
+
+    @Test
+    public void testWhyDoomed() {
+        Game mockGame = Mockito.mock(Game.class);
+        Entity mockEntity;
+        PlanetaryConditions planetaryConditions;
+        Board mockBoard = Mockito.mock(Board.class);
+        IHex mockHex = Mockito.mock(Hex.class);
+        Coords mockCoords = Mockito.mock(Coords.class);
+
+        // Trace atmosphere - Entity doomed in vacuum/trace atmosphere
+
+        planetaryConditions = new PlanetaryConditions();
+        planetaryConditions.setAtmosphere(PlanetaryConditions.ATMO_TRACE);
+        Mockito.when(mockGame.getPlanetaryConditions()).thenReturn(planetaryConditions);
+        mockEntity = Mockito.mock(Infantry.class);
+        Mockito.when(mockEntity.doomedInVacuum()).thenReturn(true);
```

Comment:
I have been writing quite a few unit tests in MekHQ and have found adding a static import of `Mockito.*` really aids in the readability of these tests _and_ saves you hundreds (if not thousands) of keystrokes. I realize existing unit tests in MM don't quite follow that approach, but I figured I would pass it along.

Reply:
Done!